tests/test_validator_integration.py

```
@@ -0,0 +1,924 @@
+
+
+class TestRemoveUnusedColumns:
+    """GLiNER uses custom data collators.  HF TrainingArguments defaults
+    remove_unused_columns=True which strips columns the collator needs.
+    Fixed: create_training_args now defaults to False, and training_cli forwards it."""
+
+    def test_default_remove_unused_columns_is_true(self):
+        """The HF default for remove_unused_columns is True."""
+        from gliner.training.trainer import TrainingArguments
+
+        args = TrainingArguments(output_dir="/tmp/_test_ruc")
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/_test_ruc"

(S108)